AdminWebsite/AdminWebsite/ClientApp/src/app/booking/participant/item/participant-item.component.ts

```diff
@@ -95,10 +95,7 @@ export class ParticipantItemComponent {
     }
 
     canEditJudge(): boolean {
-        if (!this.canEdit || this.videoHearingsService.isConferenceClosed()) {
-            return false;
-        }
-        if (this.videoHearingsService.isHearingAboutToStart()) {
+        if ((!this.canEdit || this.videoHearingsService.isConferenceClosed()) && this.videoHearingsService.isHearingAboutToStart()) {
```

Comment:
Should this not be: 
`!this.canEdit || this.videoHearingsService.isConferenceClosed() || this.videoHearingsService.isHearingAboutToStart()`